index.js

```diff
@@ -0,0 +1,192 @@
+const Manager = require("./lib/Manager");
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+const inquirer = require("inquirer");
```

Comment:
try to import any code you didn't write yourself (e.g. packages/modules) before anything else (e.g.place inquirer + fs at the top of the file)